src/lib.rs

```diff
@@ -1,3 +1,55 @@
+use std::fmt::{Debug, Formatter, Result, Write};
+
+// Allows to override the Debug implementation provided by a given type.
+pub struct Hidden<'borrow, T>(pub &'borrow T);
+
+impl<'borrow, T> From<&'borrow T> for Hidden<'borrow, T> {
+    fn from(ref_: &'borrow T) -> Hidden<'borrow, T> {
+        Hidden(ref_)
+    }
+}
+
+impl<'borrow, T> Debug for Hidden<'borrow, T>
+where
+    T: Debug,
+{
+    fn fmt(&self, f: &mut Formatter) -> Result {
+        let mut writer = HiddenWritter::new(f);
+
+        // TODO(scrabsha): handle the alternate flag:
+        // https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.alternate
+        write!(writer, "{:?}", self.0)
+    }
+}
+
+struct HiddenWritter<'fmt, 'a>(&'fmt mut Formatter<'a>);
```

Comment:
Writter → Writer

Reply:
Fixed in [#33c02de](https://github.com/iomentum/hunter2/pull/1/commits/33c02dea80dd22ec07cf3b21f06cec0eb6251668).